bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/obs/ObsListView.scala

```diff
@@ -687,6 +687,19 @@ class ObsListView(shellAdvisor:ShellAdvisor, band:Band, queueLookup: Target => U
       }
     }
 
+    // Edit the blueprint
+    def fixBlueprint(b: BlueprintBase) {
```

Comment:
I guess you tested this on the UI, right? It should open the instrument decision tree for the given blueprint